src/hooks/video/agora/useAgoraScreenShare.ts

```diff
@@ -25,7 +25,8 @@ export const useAgoraScreenShare: ReactHook<
   const [localAudioTrack, setLocalAudioTrack] = useState<ILocalAudioTrack>();
 
   const stopShare = useCallback(async () => {
-    if (!venueId || !userId || !client) return;
+    if (!venueId || !userId || !client || !localScreenTrack || !localAudioTrack)
+      return;
 
     updateTalkShowStudioExperience({
```

Comment:
Is it correct to not call `updateTalkShowStudioExperience` when `!localScreenTrack || !localAudioTrack`? The previous way would have ensured the backend data stayed in sync with the expected reality I would have thought? Whereas now I expect there is more chance of them being weirdly out of sync.

Reply:
@0xdevalias fixed in https://github.com/sparkletown/sparkle/pull/1653/commits/1a95556a6dfecdc953e99c7e694c1f6b703180f9